pgml-dashboard/src/components/star/star_controller.js

```diff
@@ -0,0 +1,13 @@
+import { Controller } from '@hotwired/stimulus'
```

Comment:
You can delete empty controllers, bundle just won't include them.

Reply:
Yes I just removed this!